ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -511,7 +511,7 @@ subjects:
     namespace: openshift-monitoring
 EOF
 
-echo "Promtail manifests created, the cluster can be found at https://grafana-loki.ci.openshift.org/explore using '{invoker=\"${OPENSHIFT_INSTALL_INVOKER}\"} | unpack' query"
+echo "Promtail manifests created, the cluster can be found at https://grafana-loki.ci.openshift.org/explore using '{invoker=\"${OPENSHIFT_INSTALL_INVOKER}\"} | unpack' query. See https://hackmd.io/zx5Av4mTQKqdL7EF0JyYBQ?view for Loki cheat sheet."
```

Comment:
What do you think about adding a link from the lens?

Reply:
Sure, that would make it more visible